Engine/Tracker/Filters/kalman_filter.py

```diff
@@ -10,15 +10,18 @@ def __init__(self, _id=None):
         self.is_active = False
         self.last_update_time = 0
         self.first_update_time = 0
-
+        self.observation_covariance = np.diag([1, 1])
+        self.observation_model = np.array([[1, 0, 0, 0],   # Position x
+                                          [0, 0, 1, 0]])  # Position y
         self.state_number = int(np.size(self.transition_model(0), 0))
-        self.observable_state = int(np.size(self.observation_model(), 0))
+        self.observable_state = int(np.size(self.observation_model, 0))
 
         self.x = np.zeros(self.state_number)
 
-        self.R = self.observation_covariance
         self.Q = self.process_covariance
         self.P = self.initial_state_covariance()
+        self.logger = logging.getLogger("KALMAN_FILTER")
```

Comment:
use the class name variable like the other one..

Reply:
je suis pas exalted avec la faction "radiant python overlord" sorry :(

Reply:
self.name était pas defined, pis les autre c genre dans le super.init, jecomprend rien a ce genre daffaire là :(((((((((

Reply:
Tu peux utiliser `self.__name__`